[Email|Mailbox] Show .ics files as attachments

gudsson opened the pull request:
# Code Changes
- Removed 'text/calendar' for disallowed content types to allow display of .ics calendar attachments
- added check to `isFileAttachment` that verifies file has a `filename` property set.  .ics files are appending two files to file list, one without a set filename.

# Screenshots:

## Before
![Screen Shot 2022-03-28 at 10 42 57 PM](https://user-images.githubusercontent.com/43771331/160535410-6b7fa12f-ec8b-4271-827e-7686d01d796f.png)

## After
![Screen Shot 2022-03-28 at 10 47 06 PM](https://user-images.githubusercontent.com/43771331/160535434-7b0e4843-4ba6-4415-aea5-9d548706bf3a.png)

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner

AaronDDM commented:
@gudsson could you rebase off main to fix your failing tests?

Also, could you possibly add a cypress test for your latest fix/update? Thanks!

gudsson commented:
> @gudsson could you rebase off main to fix your failing tests?
> 
> Also, could you possibly add a cypress test for your latest fix/update? Thanks!

Rebased and added tests.

pooja169usp reviewed (approved):
LGTM
